frontend/lib/geo-autocomplete.tsx

```diff
@@ -181,7 +181,7 @@ export class GeoAutocomplete extends React.Component<GeoAutocompleteProps, GeoAu
   }
 
   getInputProps(ds: ControllerStateAndHelpers<GeoAutocompleteItem>) {
-    return ds.getInputProps({
+    return ds.getInputProps<React.HTMLProps<HTMLInputElement>>({
```

Comment:
For some reason the typings for `getInputProps` changed to be generic, perhaps because it's possible now for the underlying `<input>` element to be any react component or something.  In any case, I just forced a type here so that the same type-checking/inference takes place as it did with the previous version of downshift.